tensorflow/core/api_def/base_api/api_def_ParallelInterleaveDataset.pbtxt

```diff
@@ -1,6 +1,60 @@
 op {
   graph_op_name: "ParallelInterleaveDataset"
   visibility: HIDDEN
+  in_arg {
+    name: "input_dataset"
+    description: <<END
+Dataset that produces a stream of arguments for the function `f`.
+END
+  }
+  in_arg {
+    name: "other_arguments"
+    description: <<END
+Additional arguments to pass to `f` beyond those produced by `input_dataset`.
+Evaluated once when the dataset is instantiated. 
+END
+  }
+  in_arg {
+    name: "cycle_length"
+    description: <<END
+Number of datasets (each created by applying `f` to the elements of
+`input_dataset`) among which the `ParallelInterleaveDataset` will cycle in a 
+round-robin fashion.
+END
+  }
+  in_arg {
+    name: "block_length"
+    description: <<END
+Number of elements at a time to produce from each interleaved invocation of a
+dataset returned by `f`.
+END
+  }
+  in_arg {
+    name: "sloppy"
+    description: <<END
+If `True`, return elements as they become available, even if that means returning
+these elements in a different order.
+If `False`, the operation of this dataset will be deterministic, but the
+dataset may block.
```

Comment:
The dataset may "block" even when sloppy is True. Instead say, that sloppy operation may result in better performance in the presence of stragglers (e.g. due to I/O).